app/models/voucher.rb

```diff
@@ -167,6 +167,7 @@ def inverted_rows(options = {})
     rows.reject!(&:canceled?) unless options[:canceled]
     rows.map do |old|
       vr = old.clone
```

Comment:
Changed to using dup instead. A quick search in the code base shows that there are clone in some other places too.